frontend/src/App.tsx

```diff
@@ -131,6 +132,18 @@ const App = (): React.ReactElement => {
                 component={ProfilePage}
                 allowedRoles={AuthConstants.ALL_ROLES}
               />
+              <PrivateRoute
+                exact
+                path={Routes.ADMIN_PAGE}
+                component={Default}
```

Comment:
should you be passing in the AdminPage component here?

Reply:
Oh yep mb